config/routes.rb

```diff
@@ -0,0 +1,61 @@
+Rails.application.routes.draw do
+
+  get "/posts", to: "posts#index"
+  get "/comments", to: "comments#index"
+  post "/posts", to: "posts#create"
```

Comment:
Really want you to implement full CRUD functionality for this app. Good start would be using `resources` for one or all of your models.